src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -94,9 +95,22 @@ class ContainerBuilder extends Container implements TaggedContainerInterface
      */
     private $usedTags = array();
 
+    private $context;
+
     private $compiled = false;
 
     /**
+     * @param ParameterBagInterface|null $parameterBag A ParameterBagInterface instance
+     * @param Context|null               $context      A Context instance, or null
```

Comment:
I kind of suspected it but isn't this more a problem on phpStorm side than anything?